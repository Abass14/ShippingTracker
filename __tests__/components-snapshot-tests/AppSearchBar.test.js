import ShallowRenderer from 'react-shallow-renderer';
import AppSearchBar from '../../src/app/components/search-bar';

it('should render SearchBar correctly', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer.render(
        <AppSearchBar />
    )
    expect(tree).toMatchSnapshot()
})