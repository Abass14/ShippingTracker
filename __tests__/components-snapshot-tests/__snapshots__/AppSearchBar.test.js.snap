// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SearchBar correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F4F2F8",
      "borderRadius": 10,
      "flexDirection": "row",
      "gap": 10,
      "height": 44,
      "paddingHorizontal": 14,
    }
  }
>
  <SearchIcon />
  <TextInput
    onChangeText={[Function]}
    style={
      [
        {
          "color": "#2F50C1",
          "flex": 1,
          "fontSize": 16,
        },
        undefined,
      ]
    }
  />
</View>
`;
